receiver/mongodbreceiver/metadata.yaml

```diff
@@ -1,5 +1,10 @@
 name: mongodbreceiver
 
+resource_attributes:
+  database:
+    description: The name of a database.
+    type: string
+
 attributes:
   database:
```

Comment:
Is it expected to have the attribute and resource_attribute be a duplicate of one another?

Reply:
Yes, this is current behavior. I submitted a separate issue to address that https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/9760